src/components/FeatureViews/AdminPortal/ActionLog.js

```diff
@@ -96,12 +113,8 @@ const ActionLog = () => {
         "sort": "",
         "clicked": false,
     });
-    const [fetchAdmins] = useLazyQuery(GET_ADMINS, {
-        "onCompleted": (data) => {
-            setUserOptions(data.admins);
-        },
-    });
-    const [logs] = useLazyQuery(GET_LOGS, {
+    const admins = useQuery(GET_ADMINS);
```

Comment:
you can combine multiple queries into 1 query. this is an example where I'm querying both admins and courses in the same query.
![Screen Shot 2020-08-05 at 8 12 50 PM](https://user-images.githubusercontent.com/12959959/89486504-15240f00-d758-11ea-95d3-4789a1973ce8.png)

Reply:
basically - you should combine your logs and your admin queries into 1 query that fetches both